pkg/cmd/server/start/master_args.go

```diff
@@ -182,6 +185,19 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 
 	dnsServingInfo := servingInfoForAddr(&dnsBindAddr)
 
+	overrideConfig := configapi.ClusterResourceOverrideConfig{}
```

Comment:
Let me connect the dots now that I have a little more time to pay attention.

The task is to get this plugin's config out to the console. That console config is provided over here https://github.com/sosiouxme/origin/blob/31c763839b1dbf4c7e6612582a9131d43952f3e0/pkg/cmd/server/origin/asset.go#L200 where it really only has the AssetConfig object to work with.

Said AssetConfig object is basically just a container for configapi.AssetConfig https://github.com/sosiouxme/origin/blob/31c763839b1dbf4c7e6612582a9131d43952f3e0/pkg/cmd/server/origin/asset_config.go which it is given by start_master.StartAPI https://github.com/sosiouxme/origin/blob/31c763839b1dbf4c7e6612582a9131d43952f3e0/pkg/cmd/server/start/start_master.go#L454

So alright, I guess I can just expand the origin.AssetConfig object https://github.com/sosiouxme/origin/blob/31c763839b1dbf4c7e6612582a9131d43952f3e0/pkg/cmd/server/origin/asset_config.go#L8 with a plugin config member and have StartAPI fill that in rather than putting it in the serializable MasterConfig.AssetConfig object. Would that be preferable?